<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoStream</class>
 <widget class="QWidget" name="videoStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>85</y>
     <width>391</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="video">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>39081</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);
font: bold 15pt &quot;Sans Serif&quot;; 
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>  Streaming Debug</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>390</width>
       <height>554</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(163, 163, 163);
font: bold 8pt &quot;Sans Serif&quot;;
color:rgb(42, 42, 42);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DB">
    <attribute name="title">
     <string>DB</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);
border-bottom-color: rgb(255, 255, 255);
font: bold 15pt &quot;Sans Serif&quot;; 
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>  DB Monitoring</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearchDB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>44</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SearchDB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsertDB">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>44</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>InsertDB Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnectDB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>ConnectDB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>391</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);
border-top-color: rgb(0, 0, 0);
font: bold 15pt &quot;Sans Serif&quot;; 
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>391</width>
       <height>521</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>389</width>
        <height>519</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>391</width>
         <height>521</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(163, 163, 163);
font:bold 8pt &quot;Sans Serif&quot;;
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <zorder>scrollArea</zorder>
    <zorder>label_6</zorder>
    <zorder>label</zorder>
    <zorder>btnSearchDB</zorder>
    <zorder>btnInsertDB</zorder>
    <zorder>btnConnectDB</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 62, 62);
font: bold 15pt &quot;Sans Serif&quot;; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  Server Monitoring</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>661</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="CAM1">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>491</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>If you want to register a new camera URL,
 right-click</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget_2</zorder>
  <zorder>tabWidget</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
